### ISSUE TITLE
When deploying a skaffold run config, I'm seeing a balloon with a message about the current context

### ISSUE THREAD

**Issue by matthewmichihara** [GoogleCloudPlatform/cloud-code-intellij#2542]
![image](https://user-images.githubusercontent.com/79073/57310419-c81a3e80-70b7-11e9-837a-c00aed79e678.png)

This should be event log only with no balloon.

**Comment by matthewmichihara** [comment]
Sarah doesn't seem to get this so now I'm wondering if something in my settings is messed up.

**Comment by ivanporty** [comment]
I think we should not use the same plugin identifier group for the messages with baloon and with the event log. Need to have "Cloud Code Notifications" and "Cloud Code Event Log" or something similar. For me, the effect is vice versa - all notifications went into event log instead of balloons.

**Comment by ivanporty** [comment]
`PluginNotificationSettings.PLUGIN_NOTIFICATION_DISPLAY_GROUP_ID` - this should only be used for balloons, and another one should fix this issue if we only use it for log messages.

**Comment by matthewmichihara** [comment]
ah ok , looking into fixing this now.

**Comment by matthewmichihara** [comment]
Just for reference, one place this shows up is in this notifications settings menu
![image](https://user-images.githubusercontent.com/79073/57320717-898f7e80-70cd-11e9-942f-0d3609a53e3f.png)

I'm going to go with `Cloud Code Notifications` and `Cloud Code Event Log` as @ivanporty suggested.

**Comment by matthewmichihara** [comment]
Looking at this more, it's probably best not to group notifications by their display style, but by feature since the intent seems to be to allow to let people configure how they want their different notifications to display (balloon | no popup | etc).

As a quick fix to this particular issue, I can split between balloon and event log, but I think it's worth thinking about this more.

Thoughts @ivanporty @etanshaul ?

**Comment by etanshaul** [comment]
good point. agree we need to think about how to group these.

> As a quick fix to this particular issue, I can split between balloon and event log

Could you elaborate? Is this what you did in https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/pull/132 ?

**Comment by matthewmichihara** [comment]
Yup thats what I did there.

**Comment by etanshaul** [comment]
Sounds good. We can expand on this, post-release, to further subdivide by feature area.

**Comment by etanshaul** [comment]
@matthewmichihara can we close this now?

**Comment by matthewmichihara** [comment]
I left it open to track subdividing notifications by feature area instead of by display type. I should rename the title of this issue. How does that sound?

**Comment by etanshaul** [comment]
ah right. sg. and thanks for updating the title.

**Comment by ivanporty** [comment]
Good idea! Could you elaborate a bit more on what are the feature groups you have in mind?

**Comment by matthewmichihara** [comment]
Open to suggestions, but to start, I'm thinking of splitting between `Cloud Code` (for most existing cloud code notifications), and `Cloud Code Build` (for the stuff currently event log only - k8s context switching)
